handler.go

```diff
@@ -253,6 +253,7 @@ func (h FilteredHttpRequestHandler) Handle(req *http.Request, metrics Metrics) (
 		_, ok := h.allowedOrigins[req.Host]
 		if !ok {
 			metrics.Fire(metricsResultTargetRequestForbidden)
+			log.Printf("TargetForbiddenError: %s", req.Host)
```

Comment:
Can we remove this log line?

Reply:
We use it to be able to identify and fix not-allowed requests for the content. It happens that some services use direct links to our S3 web wrapper service. And we wanto to log these cases and fix as time permits.

Reply:
Yeah, that's a fine use case, but it should be done in a separate PR. I also wonder if there's a better way to track these beyond logging (and then scraping the logs).

Reply:
but we could put it under if (VERBOSE). Will it work for you?

Reply:
That might work. Can we do this in a separate PR?